eclipse-collections/src/main/java/org/eclipse/collections/impl/map/mutable/UnifiedMap.java

```diff
@@ -1430,6 +1430,94 @@ public void value(K key, V value)
         return builder.toString();
     }
 
+    public boolean trimToSize()
+    {
+        if (this.table.length <= this.occupied)
```

Comment:
Same comment for UnifiedSet as well? Or we needn't do that for UnifiedSet since we do not need powers of 2.